app/controllers/pseuds_controller.rb

```diff
@@ -14,7 +14,7 @@ def load_user
   # GET /pseuds.xml
   def index
     if @user
-      @pseuds = @user.pseuds
+      @pseuds = @user.pseuds.paginate(page: params[:page], per_page: ArchiveConfig.ITEMS_PER_PAGE)
```

Comment:
Can we leave out the `per_page` argument, since `ArchiveConfig.ITEMS_PER_PAGE` is the default value?

Reply:
Oh, I suppose we can! From a consistency POV, we're already using it like this (almost) everywhere, but I guess there's no point introducing redundancy when writing _new_ code.

(I had to dig to find out where the defaults are actually set, and found them in `config/initializers/gem-plugin_config/will_paginate_config.rb`, for anyone wondering the same now.)